blazor/mention/code-snippet/item-template.razor

```diff
@@ -0,0 +1,47 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Data
+
+<SfMention TItem="EmployeeData" Query="@Query" PopupWidth="250px" suggestionCount=6>
+        <TargetComponent>
+            <div id="commentsMention" placeHolder="Type @@ and tag employee"></div>
+        </TargetComponent>
+        <ItemTemplate>
+            <span><span class='name'>@((context as EmployeeData).FirstName)</span><span class='country'>@((context as EmployeeData).Country)</span></span>
+        </ItemTemplate>
+         <ChildContent>
+             <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
+             <MentionFieldSettings Text="FirstName" Value="EmployeeID"></MentionFieldSettings>
+         </ChildContent>
+</SfMention>
+
+<style>
+    #commentsMention {
+        min-height: 100px;
+        border: 1px solid #D7D7D7;
+        width: 600px;
+        font-size: 14px;
+        padding: 8px;
+        border-radius: 4px;
+    }
+
+    div#commentsMention[placeholder]:empty:before {
+        content: attr(placeholder);
+        color: #555;
+    }
+
+    .country {
+        right: 15px;
+        position: absolute;
+    }
+</style>
+
+@code {
+
+    public class EmployeeData
+    {
+        public string FirstName { get; set; }
+        public string Country { get; set; }
+    }
+    public EmployeeData Data = new EmployeeData();
+    public Query Query = new Query();
+}
```

Comment:
Don't add the unwanted code here. check the all code snippets

Reply:
Removed the unwanted code.